[2024-06-24 12:53:33,175] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task 2021-08-24T00:00:00+00:00 [queued]>
[2024-06-24 12:53:33,287] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task 2021-08-24T00:00:00+00:00 [queued]>
[2024-06-24 12:53:33,290] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2024-06-24 12:53:33,301] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2024-06-24 12:53:33,310] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2024-06-24 12:53:33,447] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): second_task> on 2021-08-24T00:00:00+00:00
[2024-06-24 12:53:33,507] {standard_task_runner.py:52} INFO - Started process 28702 to run task
[2024-06-24 12:53:33,639] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v5', 'second_task', '2021-08-24T00:00:00+00:00', '--job-id', '806', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpsnwp5c05', '--error-file', '/tmp/tmpvheg83jz']
[2024-06-24 12:53:33,676] {standard_task_runner.py:77} INFO - Job 806: Subtask second_task
[2024-06-24 12:53:34,468] {logging_mixin.py:104} INFO - Running <TaskInstance: our_first_dag_v5.second_task 2021-08-24T00:00:00+00:00 [running]> on host 799b1e36e3f1
[2024-06-24 12:53:35,008] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-08-24T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-24T00:00:00+00:00
[2024-06-24 12:53:35,016] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2024-06-24 12:53:35,045] {bash.py:158} INFO - Running command: echo hey, I am task2 and will be running after task1!
[2024-06-24 12:53:35,126] {bash.py:169} INFO - Output:
[2024-06-24 12:53:35,135] {bash.py:173} INFO - hey, I am task2 and will be running after task1!
[2024-06-24 12:53:35,138] {bash.py:177} INFO - Command exited with return code 0
[2024-06-24 12:53:35,385] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20210824T000000, start_date=20240624T125333, end_date=20240624T125335
[2024-06-24 12:53:35,766] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-24 12:53:35,810] {local_task_job.py:146} INFO - Task exited with return code 0
